cql2pgjson/src/main/java/org/z3950/zing/cql/cql2pgjson/CQL2PgJSON.java

```diff
@@ -1007,11 +896,6 @@ private IndexTextAndJsonValues multiFieldProcessing( String index ) throws Query
     // if no json field name prefix is found, the default field name gets applied.
     String defaultJsonField = this.jsonFields.get(0);
     String finalIndex = index;
```

Comment:
The variable finalIndex is no longer needed and should be removed. Use index instead.